spec/request/service_brokers_spec.rb

```diff
@@ -90,13 +90,17 @@
   end
 
   describe 'GET /v3/service_brokers' do
+    let(:api_call) { lambda { |user_headers| get '/v3/service_brokers', nil, user_headers } }
+
     before do
       stub_request(:get, 'http://example.org/broker-url-v2/v2/catalog').
         to_return(status: 200, body: catalog('v2-broker').to_json, headers: {})
       stub_uaa_for('v2-broker')
     end
 
-    let(:api_call) { lambda { |user_headers| get '/v3/service_brokers', nil, user_headers } }
+    it_behaves_like 'paginated response', '/v3/service_brokers' do
+      let!(:resources) { (0..1).collect { VCAP::CloudController::ServiceBroker.make } }
+    end
```

Comment:
```suggestion
      let!(:resources) { Array.new(2) { VCAP::CloudController::ServiceBroker.make } }
    end
```

Not sure if this is what you are trying to do?